Userland/Libraries/LibWeb/Loader/GeneratedPagesLoader.cpp

```diff
@@ -72,4 +73,22 @@ ErrorOr<String> load_file_directory_page(AK::URL const& url)
     return TRY(String::from_utf8(generator.as_string_view()));
 }
 
+ErrorOr<String> load_about_version_page()
+{
+    // Generate HTML about version page from template file
+    // FIXME: Use an actual templating engine (our own one when it's built, preferably with a way to check these usages at compile time)
+    auto template_path = TRY(Core::Resource::load_from_uri("resource://ladybird/templates/version.html"sv))->filesystem_path();
+    auto template_file = TRY(Core::File::open(template_path, Core::File::OpenMode::Read));
+    auto template_contents = TRY(template_file->read_until_eof());
```

Comment:
:thinking:  this is opening the file twice, no? The file is opened already by `Core::Resource`. You can get its data with `Core::Resource::data()`.

Reply:
Oh you're right nice, that code structure came from before resources were a thing